app/javascript/pages/orientation/Carousel.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+    <section>
+        <b-carousel
+            :autoplay="auto"
+            :arrow="arrow"
+            :repeat="arrowBoth"
+            :arrow-hover="arrowHover"
+            :interval="interval"
+            :pause-hover="pauseHover">
+            <b-carousel-item v-for="(announcement, i) in announcements" :key="i" :id="announcement.id">
+                <section :class="`hero is-light`" style="background-color:grey-lightest">
+                    <div class="hero-body has-text-centered">
+                        <h1 class="title"  v-if="announcement.id === 'default'">{{announcement.name}}</h1>
+                        <h1 class="title"  v-else>
+                            <a :href="`/announcements/${announcement.id}`">{{announcement.name}}</a>
+                        </h1>
+                        <p class="content">{{announcement.description}}</p>
+                    </div>
+                </section>
+            </b-carousel-item>
+        </b-carousel>
+    </section>
+</template>
+
+<script>
+import { Carousel } from 'buefy';
+const fetch = require("node-fetch");
```

Comment:
Minor: for consistency, probably better to use `import` here instead of `require`.